### PR TITLE
:bomb: :lipstick:  Zombies Assignment --- Remove asn1 stuff from FAQ

### PR DESCRIPTION
**Pull request by jameshughes89**
### What

Remove the assignment 1 specific stuff from the assignment 2 FAQ

### Additional Notes

I still want to clean them up more in general, but whatever, this is a quicker and easier thing to fix.

### CODE REVIEW

File: site/assignments/tic-tac-toe/tic-tac-toe.rst [hunk 1]
```diff
@@ -1,7 +1,166 @@
 *******************************************
-I'm only taking CS to make sick video games
+I'm Only Taking CS to Make Sick Video Games
 *******************************************
```
Comment (twentylemon) [inline site/assignments/tic-tac-toe/tic-tac-toe.rst:3]: This file seems unrelated.

Comment (jameshughes89) [reply site/assignments/tic-tac-toe/tic-tac-toe.rst:3]: how the f did that sneak in there.
